### PR TITLE
chore: add some more hints for playground

### PR DESCRIPTION
**Pull request by xxchan**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?
close https://github.com/risingwavelabs/risingwave/issues/4331

### CODE REVIEW

File: src/meta/src/lib.rs [hunk 1]
```diff
@@ -120,7 +120,7 @@ pub struct MetaNodeOpts {
     /// After specified seconds of idle (no mview or flush), the process will be exited.
     /// It is mainly useful for playgrounds.
     #[clap(long)]
-    dangerous_max_idle_secs: Option<u64>,
+    pub dangerous_max_idle_secs: Option<u64>,
```
Comment (tabVersion) [inline src/meta/src/lib.rs:123]: Just curious, what this field for?

Comment (xxchan) [reply src/meta/src/lib.rs:123]: https://github.com/risingwavelabs/risingwave/pull/3444

File: Makefile.toml [hunk 2]
```diff
@@ -397,6 +397,9 @@ script = '''
 
 set -e
 
+
+echo "$(tput bold)$(tput setaf 9)RisingWave playground SHOULD NEVER be used in benchmarks and production environment!!!
```
Comment (skyzh) [inline Makefile.toml:401]: shall we move this to `risingwave_cmd_all`? Users can also start playground by using that binary.

**Review by skyzh — APPROVED** [review]

File: Makefile.toml (hunk 2 above)
Comment (xxchan) [reply Makefile.toml:401]: I had also thought about it. The benefit of putting it here is that it is before compiling and running, so it is very noticable without being mixed in logs... 🤪

What about putting it at both places?

Comment (skyzh) [reply Makefile.toml:401]: Sounds good.

Comment (xxchan) [reply Makefile.toml]: Moved to cmd & added a `sleep` to make it more noticable. Is that OK?

image

**Review by skyzh — APPROVED** [review]

**Pull request merged**